bw2io/importers/ecospold1.py

```diff
@@ -82,18 +93,71 @@ def __init__(
             )
         )
 
-
 class NoIntegerCodesEcospold1Importer(SingleOutputEcospold1Importer):
+    """
+    An importer class that deletes integer codes from ecospold1 datasets.
+    
+    Parameters
+    ----------
+    SingleOutputEcospold1Importer : class
+        The base importer class.
+    
+    Attributes
+    ----------
+    strategies : list
+        A list of strategies that the importer applies to process the dataset.
+    
+    Returns
+    -------
+    None
+    """
     def __init__(self, *args, **kwargs):
+        """
+        Initialize NoIntegerCodesEcospold1Importer.
+        
+        Parameters
+        ----------
```

Comment:
@cmutel does it make sense to include the `args`, `kwargs` in these cases?

Reply:
@michaelweinold @cmutel Actually, as I know it's a good practice to include *args and **kwargs in NumPy docstrings. It also makes the documentation more complete and informative, especially for users who might be unfamiliar with the codebase.

Also, including *args and **kwargs is part of the NumPy docstring standard.